css/cssom/cssstyledeclaration-setter-form-controls.html

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
+<title>CSSOM test: no side effects from setting "height"</title>
+<link rel="help" href="https://drafts.csswg.org/cssom/#dom-cssstyledeclaration-setproperty">
+<link rel="help" href="https://bugs.webkit.org/show_bug.cgi?id=107380">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<p>
+Historically, the Apple Safari web browser has added an "intrinsic margin" to
+form controls which do not specify a `height`. It removed this margin following
+modification of the `height`. <a
+href="https://bugs.webkit.org/show_bug.cgi?id=107380">This non-standard
+behavior was identified as a source of confusion for web developers.</a>
+</p>
+
+<script>
+test(function() {
+  var element = document.createElement('input');
```

Comment:
The tests are very similar, only varying on the element and attribute. Can you make it more DRY?

For example,
```
const tests = [['input', 'text'], ..., ['button']];
for (const t of tests) {
  var [tagName, type] = t;
  test(...);
}
```